utils/config/wallets/BaseDotsamaWallet.ts

```diff
@@ -107,7 +107,9 @@ export class BaseDotsamaWallet implements Wallet {
       callback(undefined)
       return null
     }
-
+    if (!this._extension.accounts.subscribe) {
+      return null
+    }
```

Comment:
What's this? does it fix something?

Reply:
yes, the clv extension wallet should have implemented the subscribe function. : )